harvard-lts/CURIOSity
# Remove breadcrumbs navigation

**Remove breadcrumbs navigation**
* * *

**JIRA Ticket**: [LTSMAINT-856](https://at-harvard.atlassian.net/browse/LTSMAINT-856)

# What does this Pull Request do?
Removes the breadcrumb navigation completely from the markup so that screenreaders  do not read an empty nav.

# How should this be tested?

To see what it is currently doing:
- Navigate to any collection homepage, like this one: https://curiosity.lib.harvard.edu/photographs-of-black-theater-by-alix-jeffry
- Inspect the code and search for the .breadcrumbs-container div element (this should be right below the header element and before the main element); this is what we want to remove

To test the fix:
- Spin up this branch and navigate to any collection homepage
- Inspect the code and confirm that the .breadcrumbs-container element no longer appears

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@VanessaVenti @enriquediaz
